css/animate.css

```diff
@@ -0,0 +1,1579 @@
+@charset "UTF-8";
```

Comment:
Don't bundle any CSS libraries. If we need one, we must use an npm package for it.